distiller/settings/base.py

```diff
@@ -105,7 +113,7 @@
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
 DATABASES = {
-    'default': dj_database_url.config(default='postgres://postgres:postgres@db/distiller')
+    'default': dj_database_url.config(default='postgres://postgres:postgres@127.0.0.1/distiller')
```

Comment:
I haven't tested, but does changing the db host still work in Docker-land?

Reply:
I wasn't able to get it to work immediately, so punted for the moment. I'll try and revisit - we probably need to tweak the network config in `docker-compose.yml`.